tests/transport_test.go

```diff
@@ -0,0 +1,118 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pkg/errors"
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+
+	"k8s.io/api/core/v1"
+)
+
+var _ = Describe("Transport Tests", func() {
+
+	const (
+		annEndpoint = "cdi.kubevirt.io/storage.import.endpoint"
+		annSecret   = "cdi.kubevirt.io/storage.import.secretName"
+
+		fileHostName   = "cdi-file-host" // deployment and service name
+		fileHostNs     = "kube-system"   // deployment and service namespace
+		fileHostUname  = "admin"         // http and s3 username
+		fileHostPasswd = "password"      // http and s3 password
+
+		httpPortName = "http"
+		s3PortName   = "s3"
+
+		targetFile = "tinyCore.iso"
+		bucket     = "images"
+	)
+
+	var (
+		httpEp, s3Ep, ns string
+		fileHostService  *v1.Service // persistent file host service, routes to http and s3 ports
+	)
+
+	f, err := framework.NewFramework("", framework.Config{false})
+	if err != nil {
```

Comment:
These 3 lines are necessary after _every_ call to `NewFramework`. Would you like to see an additional constructor added named `NewFrameworkOrDie` that calls `Fail` if a new framework error occurs? If not that's fine too...

Reply:
An `orDie` wrapper would be a good idea but I think it'd be better done in a separate PR since other tests are going to be affected by it.  

Also, just fyi, you can't use `Expect` outside of BeforeEach/After or It blocks.  Calling it causes a panic.

Reply:
Ok, so `Expect` cannot be used in a `Describe`.... then the `NewFrameworkOrDie` func would need to call `Fail` instead.